packages/flutter/lib/src/widgets/system_context_menu.dart

```diff
@@ -84,7 +84,7 @@ class SystemContextMenu extends StatefulWidget {
         ),
       ),
       items: items ?? getDefaultItems(editableTextState),
-      onSystemHide: () => editableTextState.hideToolbar(false),
+      onSystemHide: editableTextState.hideSystemToolbar,
```

Comment:
Nit: I think hideSystemToolbar should be named something with "handle" in the name, like "handleSystemHideToolbar": https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#naming-rules-for-typedefs-and-function-variables